blog/tag/seo.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: tag
+tag: seo
+permalink: /blog/tag/seo/
+redirect_from:
```

Comment:
You'll need to add the tag in the `/_data/tags.yml` file as well for it to show.